auth0_flutter/FAQ.md

```diff
@@ -72,6 +72,57 @@ You still need to call `logout()` on Android, though, as `useEphemeralSession` i
 
 > ⚠️ `useEphemeralSession` relies on the `prefersEphemeralWebBrowserSession` configuration option of `ASWebAuthenticationSession`. This option is only available on [iOS 13+](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession/3237231-prefersephemeralwebbrowsersessio), so `useEphemeralSession` will have no effect on iOS 12. To improve the experience for iOS 12 users, see the approach described below.
 
+### Use `SFSafariViewController`
```

Comment:
Fixed in f88d22d1eb2a0f508bc597423792c4940e760f21, let me know if there's a neater way to do this

Reply:
I've added this in https://github.com/auth0/auth0-flutter/pull/198/commits/74ec2a86ea0a34fbdfc5cac2f42414c37bb7095e

I omitted the note around `clearSession` but let me know if it should be added but with different wording (and let me know what the correct wording would be).

> Note that with `useEphemeralSession()` you don't need to call `clearSession(federated:)` at all. Just clearing the credentials from the app will suffice. What `clearSession(federated:)` does is clear the shared session cookie, so that in the next login call the user gets asked to log in again. But with `useEphemeralSession()` there will be no shared cookie to remove.

Reply:
`clearSession` is `logout` in the Flutter SDK. I'd suggest adding it, otherwise people call it.

Reply:
Add in https://github.com/auth0/auth0-flutter/pull/198/commits/9dfd6dae9623844482a5c2a0b5e32ef29c994444